linux_os/guide/system/bootloader-grub2/non-uefi/grub2_password/rule.yml

```diff
@@ -26,11 +26,13 @@ description: |-
     NOTE: the bootloader superuser account and password MUST differ from the
     root account and password.
     {{% endif %}}
+    {{% if "ubuntu" in product %}}
```

Comment:
Is the text change also valid for other products?
Now the text is only applicable to Ubuntu while before it was applicable to all products.

Reply:
Actually Ubuntu only uses `update-grub`, just like the UEFI case.